shopfloor/services/cluster_picking.py

```diff
@@ -564,6 +617,9 @@ def _scan_line_by_location(self, picking, move_line, location):
         return self._response_for_scan_destination(move_line)
 
     def scan_destination_pack(self, picking_batch_id, move_line_id, barcode, quantity):
+        # TODO rewrite
```

Comment:
Like something is still lacking here?

Reply:
I think this is a left over comment because all the quantity increments are made in the _scan_line_by.... and it should be removed.